lms/templates/courseware/progress.html

```diff
@@ -78,6 +78,26 @@ <h4 class="hd hd-4 title">${certificate_data.title}</h4>
                 </div>
                 %endif
 
+                %if enrollment_mode == CourseMode.AUDIT and upgrade_link:
```

Comment:
1. Would another name for `store_upgrade_cookie` be `upgrade_available`, or are these not the same?
2. Would it make sense to use `upgrade_available` instead of `enrollment_mode == CourseMode.AUDIT`, to keep visibility in sync with the earlier code?
I'm not clear if there are cases where we want (or don't want) the cookie, but we still want to show this.